changelog/fragments/3173-bugfix.yaml

```diff
@@ -0,0 +1,5 @@
+entries:
+  - description: >
+      Prevent the ansible-operator from mangling variables containing the uppercase representations of special words (e.g IP, HTTP, etc).
+    kind: "bugfix"
+    breaking: false
```

Comment:
I would actually flag this as a potentially breaking change, as the parameters being passed into the playbook are potentially changing